src/main/java/com/opinta/controller/ShipmentController.java

```diff
@@ -137,4 +137,16 @@ public ShipmentController(ShipmentService shipmentService, PDFGeneratorService p
             return new ResponseEntity<>(deleteOnErrorLogEndpoint(Shipment.class, uuid, e), NOT_FOUND);
         }
     }
+
+    @DeleteMapping("{uuid}/shipment-group")
+    public ResponseEntity<?> removeShipmentGroupFromShipment(@PathVariable UUID uuid, @RequestParam UUID token) {
+        try {
+            User user = userService.authenticate(token);
+            return new ResponseEntity<>(shipmentService.removeShipmentGroupFromShipment(uuid, user), OK);
+        } catch (AuthException e) {
+            return new ResponseEntity<>(deleteOnErrorLogEndpoint(Shipment.class, uuid, e), UNAUTHORIZED);
```

Comment:
Plz, create method deleteFieldOnErrorLogEndpoint(1, 2, 3, 4)
You should call it like this
deleteFieldOnErrorLogEndpoint(Shipment.class, ShipmentGroup.class, uuid, e)
It should produce message like this
Can't remove ShipmentGroup from Shipment {uuid}. {e.getMessage()}